### DISCUSSION TITLE
Copilot icon not appearing (VSCode extension)

### DISCUSSION THREAD

**Discussion by FC5570** [community/community#12284] [Copilot Conversations] [answered]
Currently using the [VSCode extension for Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot), however, the icon at the bottom right corner is not visible. This stopped appearing randomly a few months ago. 

![](https://ss.fc5570.me/0kZJ)
![](https://ss.fc5570.me/W2Mx)

Any fix for this?

**Comment by davecheney** [comment]
Which version of vscode are you using? Which version of the copilot extension are you using?

**Reply by FC5570** [reply]
Copilot v1.7.5185
VSCode v1.65.0 (insider)

**Comment by eliozashvili** [comment]
I have the same issue. Let me know if there will be any solution to this

**Comment by davecheney** [comment]
Could you both confirm for me 

1. you’re using vscode insiders from the vscode  website. Specifically you are not using one of the forks such as OSS Code or VSCodium. 
2. Can you try the non -insiders version and verify if the problem is the copilot extension or -insiders

**Reply by eliozashvili** [reply]
It was fixed with the latest update. Thanks <3

**Comment by AdrianLuk** [comment]
This is happening to me too.

Copilot v1.7.5209
VSCode 1.65.0(system setup)

![image](https://user-images.githubusercontent.com/26731703/156690112-deb87f47-6718-4ee6-b940-bbf62f0be218.png)

**Comment by MoKhajavi75** [comment]
Hey!

Same here:

![x](https://user-images.githubusercontent.com/25561893/156728200-30158fc0-36d8-47d2-a3cf-116296c61afe.jpg)

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Hey everyone,

This should be fixed with the latest stable version of VSCode (1.65.0) and just released the version of Copilot. Thanks a lot for all the reports!

**Reply by FC5570** [reply]
Yep. Working fine again. Thanks.

**Reply by eliozashvili** [reply]
Thank you. Everything now works fine.
